sphinx/types.rst

```diff
@@ -1,104 +1,170 @@
+the "freq" field. Users just have to ensure that the columns are already using the specified units,
+and that each column is already using the listed data type or can be converted to that type.
+
+MERRA-2
 -------
 
-Operational Data
-^^^^^^^^^^^^^^^^
-
-PlantData.scada
-
-==================== ====================
- Field Name           Data Type
-==================== ====================
- time                 datetime64[ns]
- id                   string
- power_kw             float64
- windspeed_ms         float64
- winddirection_deg    float64
- status_label         string
- pitch_deg            float64
- temp_c               float64
-==================== ====================
-
-PlantData.meter
-
-==================== ====================
- Field Name           Data Type
-==================== ====================
- time                 datetime64[ns]
- power_kw             float64
- energy_kw            float64
-==================== ====================
-
-
-PlantData.tower
-
-==================== ====================
- Field Name           Data Type
-==================== ====================
- time                 datetime64[ns]
- id                   float64
-==================== ====================
-
-PlantData.curtail
-
-==================== ====================
- Field Name           Data Type
-==================== ====================
- time                 datetime64[ns]
- curtailment_pct      float64
- availability_pct     float64
- net_energy           float64
-==================== ====================
-
-
-PlantData.status
-
-==================== ====================
- Field Name           Data Type
-==================== ====================
- time                 datetime64[ns]
- id                   string
- status_id            int64
- status_code          int64
- status_text          string
-==================== ====================
-
-
-PlantData.asset
-
-==================== ====================
- Field Name           Data Type
-==================== ====================
- id                   string
- latitude             float64
- longitude            float64
- rated_power_kw       float64
- type                 string
-==================== ====================
-
-
-Reanalysis Products
-^^^^^^^^^^^^^^^^^^^
-
-Reanalysis products are included as Plant Data objects and, regardless
-of data source, have a standardized set of field names and types (see below).
-That said, the data sources are obviously different, as are the methods use to
-calculate these standard fields from the raw datasets. These methods are described here.
-
-PlantData.reanalysis.product["merra2"]
-
-MERRA-2 data are based on the single-level diagnostic data available here:
-
-https://disc.gsfc.nasa.gov/datasets/M2T1NXSLV_V5.12.4/summary?keywords=%22MERRA-2%22
+Data are based on the single-level diagnostic data available here: https://disc.gsfc.nasa.gov/datasets/M2T1NXSLV_V5.12.4/summary?keywords=%22MERRA-2%22
```

Comment:
I'm not sure what the discussions about the "single-level diagnostic data" and corresponding links are for since we don’t require any specific heights or sources of reanalysis data, as long as variables are provided for wind speed, wind direction, temperature, etc.

Reply:
I think this whole section needs to be rewritten, to be completely honest, but I was hoping someone else might take that piece.